add-gauge-report-designer.md

```diff
@@ -0,0 +1,38 @@
+---
+title: How to Add a Dynamic Gauge in Telerik Report Designer
+description: Add a Gauge to a Report with Telerik Report Designer
+type: how-to
+page_title: Add a Gauge in Telerik Report Designer
+slug: add-dynamic-gauge-report-designer
+position: 
+tags: 
+ticketid: 1473889
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td></td>
```

Comment:
The version is missing. If this relates to all versions of Telerik Reporting, you can simply add "all"

Reply:
Added.